src/support/time.c

```diff
@@ -81,6 +81,7 @@ __wt_tsc_to_nsec(WT_SESSION_IMPL *session, uint64_t end, uint64_t begin)
 {
 	double tsc_diff;
 
+	WT_UNUSED(session);
```

Comment:
Is there any reason to pass `session` as an argument? Could probably not pass it through the `WT_TSCDIFF_NS` macro, if there's a reason that macro should take it as an argument.